spec/models/student_loans/permitted_parameters_spec.rb

```diff
@@ -0,0 +1,77 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe StudentLoans::PermittedParameters do
+  let(:claim) { create(:tslr_claim) }
+  let(:permitted_parameters) { described_class.new(claim) }
+
+  describe "params" do
+    context "when no responses have come from verify" do
+      it "includes all parameters" do
+        expect(permitted_parameters.params).to eq([
```

Comment:
Rather than listing out all the keys explicitly, this test will be less brittle (i.e. need updating every time we change the attributes) if we used the `StudentLoans::PermittedParameters::PARAMETERS ` constant here.